src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py

```diff
@@ -12260,8 +12266,9 @@ def resample(
 
         Returns
         -------
-        SnowflakeQueryCompiler
-            Holds an ordered frame with the result of the resample operation.
+        SnowflakeQueryCompiler or collections.defaultdict[Hashable, list]
+            Holds an ordered frame with the result of the resample operation, or a dictionary
+            if resample_method is 'indices'.
```

Comment:
```suggestion
            or a dictionary if resample_method is 'indices'.
```

Reply:
This is just to make it clear that we are not returning a SnowflakeQueryCompiler that holds a dictionary.